app/models/setting/provisioning.rb

```diff
@@ -29,7 +29,7 @@ def self.default_settings
       self.set('access_unattended_without_build', N_("Allow access to unattended URLs without build mode being used"), false, N_('Access unattended without build')),
       self.set('manage_puppetca', N_("Foreman will automate certificate signing upon provision of new host"), true, N_('Manage PuppetCA')),
       self.set('ignore_puppet_facts_for_provisioning', N_("Stop updating IP address and MAC values from Puppet facts (affects all interfaces)"), false, N_('Ignore Puppet facts for provisioning')),
-      self.set('ignored_interface_identifiers', N_("Ignore interfaces that match these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"), ['lo', 'usb*', 'vnet*', 'macvtap*', '_vdsmdummy_', 'veth*', 'docker*'], N_('Ignore interfaces with matching identifier')),
+      self.set('ignored_interface_identifiers', N_("Ignore interfaces that match these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"), ['lo', 'usb*', 'vnet*', 'macvtap*', '_vdsmdummy_', 'veth*', 'docker*', 'tap*', 'qbr*', 'qvb*', 'qvo*', 'qr-*', 'qg-*', 'br*', 'vlinuxbr*', 'vovsbr*'], N_('Ignore interfaces with matching identifier')),
```

Comment:
You bet!

Reply:
I guess the houndci issue should be resolved before it bites us :)

I don't mind extending the list, but I hope to get some discussion around http://projects.theforeman.org/issues/19244 so that we don't have to keep this list growing forever, as there are a lot of permutations I'm sure we don't have in the list

Reply:
Matching br* seems invalid. You probably want to match br-* 
On the other side I opened http://projects.theforeman.org/issues/21104 to recognize br-.* correctly as bridges which has a contrary intention.

Reply:
Yeah, let me split that line into multiple lines then for now. That NIC code definitely needs a refresh.

Ntr0 thank, I merged https://github.com/theforeman/foreman/pull/4872 and I will remove br* from this patch completely.